tracer/test/benchmarks/Benchmarks.Trace/AppSecBodyBenchmark.cs

```diff
@@ -90,26 +58,28 @@ static AppSecBodyBenchmark()
 #else
             httpContext = new DefaultHttpContext();
 #endif
-
         }
 
         [Benchmark]
         public void AllCycleSimpleBody() => ExecuteCycle(new { });
 
         private void ExecuteCycle(object body)
         {
-            security.InstrumentationGateway.RaiseBodyAvailable(httpContext, new Span(new SpanContext(1, 1), DateTimeOffset.UtcNow), body);
-#if NETFRAMEWORK
-            var context = httpContext.Items["waf"] as IContext;
-            context?.Dispose();
-            httpContext.Items["waf"] = null;
-#else
+            var span = new Span(new SpanContext(1, 1), DateTimeOffset.UtcNow);
+#if !NETFRAMEWORK
```

Comment:
Yes! Figured out that the benchmarks werent actually working on master, because the path to native files hadn't been updated and we were still pointing to `dd-tracer-home`... so appsec was disabled and the event wasn't raised at all.. as it was a lazy func and the event never triggered, the actual body extraction never took place, we're just now getting real numbers. 
This [draft PR](https://github.com/DataDog/dd-trace-dotnet/pull/3552) proves it, just fixing the benchmarks paths without any refactoring shows much slower benchmarks